### PR TITLE
ICMSLST-2755 -  Migrate Reports

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: data_migration/management/commands/config/run_order/files.py [hunk 1]
```diff
@@ -372,6 +372,16 @@
             "secure_lob_ref_id": DEFAULT_SECURE_LOB_REF_ID,
         },
     ),
+    QueryModel(
+        queries.schedule_reports,
+        "Schedule Reports",
```
Comment (marcuspp) [inline data_migration/management/commands/config/run_order/files.py:377]: Not sure about this one being here as these aren't files

File: data_migration/queries/reports.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+"""
+
+
+schedule_reports = """
+SELECT
+    rs.id AS id,
+    rr."DOMAIN" AS report_domain,
+    rs.title,
+    rs.DESCRIPTION AS notes,
+    rr.PARAMETER_XML as parameters_xml,
```
Comment (marcuspp) [inline data_migration/queries/reports.py:33]: The xml is no longer needed to retrieve the parameters but thought leaving it here for reference was sensible.

File: web/reports/utils.py [hunk 3]
```diff
@@ -116,6 +126,8 @@ def format_parameters_used(schedule_report: ScheduleReport) -> dict[str, str]:
                 if value
                 else "All"
             )
+        elif desc == "is_legacy_report":
+            value = "Yes" if value else "No"
```
Comment (mikeylockwood) [inline web/reports/utils.py:130]: SO is_legacy_report won't be populated or will be False for V2 generated reports?

File: data_migration/management/commands/config/run_order/files.py (hunk 1 above)
Comment (mikeylockwood) [reply data_migration/management/commands/config/run_order/files.py:377]: I think it's fine there

**Review by mikeylockwood — APPROVED** [review]

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**